css/style.css

```diff
@@ -131,105 +135,342 @@ li {
+  margin-top: 0;
+  margin-left: 24px;
+}
+
+.multi-post-title {
+  margin-left: 24px;
+  margin-top: 24px;
+  font-family: "Crete Round", sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+}
+
+.support-text-multi-post {
+  margin: 12px 24px 0 24px;
+  justify-self: left;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+}
+
+.tags-container {
+  margin-left: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-items: left;
+}
+
+.tags-container .tag-title {
+  padding: 10px 12px 10px 12px;
+  color: #091e42;
+  text-decoration: none;
+  border: 1px solid #8993a4;
+  height: 20px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.tags-container .tag-title a {
+  color: #091e42;
+  text-decoration: none;
+}
+
+.tag-cards-work-container .tag-title a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.button-see-project {
+  margin-left: 24px;
+  padding: 12px;
+  width: 135px;
+  height: 48px;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.08em;
+  text-align: center;
+  background: #ff6b00;
+  color: #fff;
+  border: none;
+  cursor: pointer;
+}
+
+.works-area-cards {
+  margin-top: 24px;
+  width: 375px;
+  height: 2436px;
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+  justify-content: center;
+}
+
+.cards-work-container {
+  width: 327px;
+  height: 386px;
+  margin-top: 24px;
+  display: flex;
+  flex-direction: column;
+  background-image: url("../images/cards-work-image.png");
+  background-size: cover;
+}
+
+.cards-work-title {
+  margin-top: 84px;
+  margin-left: 24px;
+  height: 81px;
+  width: 295px;
+  font-family: "Crete Round", sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+  color: #fff;
+}
+
+.support-text-cards-work {
+  margin-top: 12px;
+  margin-left: 24px;
+  height: 87px;
+  width: 295px;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #fff;
 }
+
+.tag-cards-work-container {
+  width: 236px;
+  height: 40px;
+  margin-left: 24px;
+  margin-top: 12px;
+  margin-bottom: 22px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.tag-cards-work-container .tag-title {
+  margin: 0 12px 0 0;
+  padding: 10px 12px 10px 12px;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  text-align: center;
+  color: #fff;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.button-cards-work {
+  padding: 12px 113.5px 12px 113.5px;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.04em;
+  text-align: center;
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  cursor: pointer;
+}
```

Comment:
- [Required] Good job implementing the styling for your buttons, however, you forgot to implement the styles for the different button states e.g `active`, `hover`, and `disabled`. Please see the screenshot below and rectify this.

## Button styling for different states
![Screenshot from 2023-03-24 01-45-26](https://user-images.githubusercontent.com/16392046/227381290-467e4a75-b303-4a05-98cb-2a5f32bdbd66.png)